doc/microsite/src/main/tut/distage/distage-testkit.md

```diff
@@ -3,10 +3,265 @@ distage-testkit
 
 @@toc { depth=2 }
 
-### Testkit
+### Quick Start
 
-[distage Example Project](https://github.com/7mind/distage-example) project shows how to use `distage-testkit`, there's also
-an overview in the [Hyper-pragmatic Pure FP Testing with distage-testkit](https://www.youtube.com/watch?v=CzpvjkUukAs) talk. 
+For effect types of `F[_]`, `F[+_, +_]` and `F[-_, +_, +_]` the `distage-testkit` library provides
+[ScalaTest](https://www.scalatest.org/) base classes. These provide an interface similar to
+ScalaTest's [`WordSpec`](http://doc.scalatest.org/3.1.0/org/scalatest/wordspec/AnyWordSpec.html).
+However, distage `SpecScalatest`s have additional capabilities. Such as first class support for
+effect types and dependency injection.
+
+Usage of `distage-testkit` generally follows these steps:
+
+1. Select a base class corresponding to the effect type:
+  - `F[_]` - @scaladoc[`DistageSpecScalatest`](izumi.distage.testkit.scalatest.DistageSpecScalatest)
```

Comment:
I think you may add the `No effect type / Identity` case here as well (which would be `DistageSpecScalatest[Identity]`)